zanata-client-commands/src/main/java/org/zanata/client/commands/pull/PullOptionsImpl.java

```diff
@@ -166,6 +167,16 @@ public boolean isContinueAfterError() {
     }
 
     @Override
+    public int getMinDocPercent() {
+        return this.minDocPercent;
+    }
+
+    @Option(name = "--min-doc-percent", aliases = "-m", usage = "When specified, will only pull down translation files that have translated or better statistics above the specified percentage")
```

Comment:
If you really want to talk in terms of our states, I think you should use
something like Translated and Approved, or "translated" and "approved" (to
make it clear that we are talking specifically about the Zanata states and
their meanings), not just 'translated or better' without quotes.

But I don't think we need to.

On 19 May 2015 at 13:53, Patrick Huang <notifications@github.com> wrote:

> In
> zanata-client-commands/src/main/java/org/zanata/client/commands/pull/PullOptionsImpl.java
> <https://github.com/zanata/zanata-client/pull/63#discussion_r30567628>:
>
> > @@ -166,6 +167,16 @@ public boolean isContinueAfterError() {
> >      }
> >
> >      @Override
> > +    public int getMinDocPercent() {
> > +        return this.minDocPercent;
> > +    }
> > +
> > +    @Option(name = "--min-doc-percent", aliases = "-m", usage = "When specified, will only pull down translation files that have translated or better statistics above the specified percentage")
>
> --min-doc-translated-percent is clearer but once we implement the "pull
> approved only in version requiring review" feature, this will become
> confusing. Although it's our internal state, but if users don't know the
> meaning they probably don't know how to use that feature.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-client/pull/63/files#r30567628>.
>